src/transformers/training_args.py

```diff
@@ -468,6 +468,15 @@ class TrainingArguments:
     )
     logging_first_step: bool = field(default=False, metadata={"help": "Log the first global_step"})
     logging_steps: int = field(default=500, metadata={"help": "Log every X updates steps."})
+    logging_nan_inf_filter: str = field(
+        default=False,
+        metadata={
+            "help": (
+                "Filter nan and inf losses for logging. "
+                "Note this flag only affects the logging output and not the optimization step."
```

Comment:
No need to be this long here, but there should be a proper documentation above that can be longer!